src/textual/css/stylesheet.py

```diff
@@ -409,6 +415,12 @@ def apply(
         # Collect the rules defined in the stylesheet
         node._has_hover_style = False
         node._has_focus_within = False
+
+        # Rules that may have an initial value
+        initial: set[str] = set()
+        # Rules in DEFAULT_CSS with an initial value
```

Comment:
See my comment above

```suggestion
        # Rules in DEFAULT_CSS set to the special value `initial`
```